File: 백준/Bronze/9498. 시험 성적/시험 성적.js
const input = require('fs').readFileSync('/dev/stdin').toString().trim().split(' ');

const score = parseInt(input[0]);

if (score >= 90 && score <= 100) {
  console.log('A');
} else if (score >= 80) {
  console.log('B');
} else if (score >= 70) {
  console.log('C');
} else if (score >= 60) {
  console.log('D');
} else {
  console.log('F');
}

File: 백준/Bronze/9498. 시험 성적/시험 성적.kt
fun main() {
    val score = readln().trim().toInt()
    
    when (score) {
        in 90..100 -> println("A")
        in 80..89 -> println("B")
        in 70..79 -> println("C")
        in 60..69 -> println("D")
        else -> println("F")
    }
}